code/jose/Python_Labs/lab02.py

```diff
@@ -5,8 +5,27 @@
 # hundreds_digit = x//100
 # print(hundreds_digit)
 
+random_numbers = {20: 'twenty',
```

Comment:
[Lab 2: Number to Phrase](https://github.com/PdxCodeGuild/class_swordfish/blob/main/1%20Python/labs/02%20Number%20To%20Phrase.md)

* Need a space between hundreds word and tens word.
* Sample output:
```
PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
Enter a number: 104
onehundred four
PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
Enter a number: 405
fourhundred five
PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs> python .\lab02.py
Enter a number: 803
eighthundred three
PS C:\Users\Bruce\Programming\class_swordfish\code\jose\Python_Labs
```